permissions/plugin-multibranch-scan-webhook-trigger.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "multibranch-scan-webhook-trigger"
+github: "jenkinsci/multibranch-scan-webhook-trigger-plugin"
+paths:
+- "igalg/jenkins/plugins/multibranch-scan-webhook-trigger"
```

Comment:
This path is incorrect. You have not set a specific `groupId` in your pom, so you will inherit the parent pom `groupId` which is `org.jenkins-ci.plugins`, so your path would be `org/jenkins-ci/plugins/multibranch-scan-webhook-trigger`

Reply:
Oooooops,
fixed in the repo